ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -203,7 +203,1503 @@ public final class GlobalTokens {
         }
     }
 
+    // MARK: - SharedColors
+
+    public enum SharedColorSets: CaseIterable {
+        case darkRed
```

Comment:
nit: I believe the order here was probably intentional so that the demo controller shows the gradients in order. But shouldn't this also be in alphabetical order?

Even in the demo controller it seems easier to find if they're ordered.